README.md

```diff
@@ -242,7 +245,7 @@ For full list refer to CMakeLists.txt
 * `ONNX_USE_PROTOBUF_SHARED_LIBS` should be ON or OFF.  
 **Default**: ONNX_USE_PROTOBUF_SHARED_LIBS=OFF USE_MSVC_STATIC_RUNTIME=0
 `ONNX_USE_PROTOBUF_SHARED_LIBS` determines how onnx links to protobuf libraries.  
-    - When set to ON - onnx will dynamically link to protobuf shared libs, PROTOBUF_USE_DLLS will be defined as described [here](https://github.com/protocolbuffers/protobuf/blob/master/cmake/README.md#dlls-vs-static-linking), Protobuf_USE_STATIC_LIBS will be set to OFF and `USE_MSVC_STATIC_RUNTIME` must be 0.  
+    - When set to ON - onnx will dynamically link to protobuf shared libs, PROTOBUF_USE_DLLS will be defined as described [here](https://github.com/protocolbuffers/protobuf/blob/main/cmake/README.md#dlls-vs-static-linking), Protobuf_USE_STATIC_LIBS will be set to OFF and `USE_MSVC_STATIC_RUNTIME` must be 0.  
```

Comment:
Are you sure that the link still work? Did protobuff folks also switch to using main instead of master?

Reply:
Thank you so much for the good catch! Protobuf repo hasn't moved the branch yet. Just corrected.